addons/mail/static/src/core/common/composer.js

```diff
@@ -359,7 +359,7 @@ export class Composer extends Component {
         }
         const suggestions = this.suggestion.state.items.suggestions;
         switch (this.suggestion.state.items.type) {
-            case "Partner":
+            case "PartnerAndRole":
```

Comment:
Is there any need to rename this?

Reply:
no, just making it clear, but I can revert it if it is not preferred